packages/venia-ui/lib/components/QuantityStepper/QuantityStepper.js

```diff
@@ -0,0 +1,94 @@
+import React, { Fragment } from 'react';
+import { useIntl } from 'react-intl';
+import { Minus as MinusIcon, Plus as PlusIcon } from 'react-feather';
+import { useQuantity } from '@magento/peregrine/lib/talons/CartPage/ProductListing/useQuantity';
+
```

Comment:
Will need to move this talon as well, and apply the re-export and alias pattern.

`export { useQuantityStepper as useQuantity } from '../QuantityStepper/useQuantityStepper'`